atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -66,7 +66,11 @@ def get_koji(workflow):
 
 
 def get_pnc(workflow):
-    return get_value(workflow, 'pnc', NO_FALLBACK)
+    try:
+        value = get_value(workflow, 'pnc', NO_FALLBACK)
+        return value
+    except KeyError:
```

Comment:
I think this should be in `PNCUtil`. Or in that lazy property (in such case, PNCUtil can receive it as param instead of workflow)

Reply:
updated.